parsers/mysql/mysql_test.go

```diff
@@ -96,10 +99,27 @@ var sqds = []slowQueryData{
 			},
 		},
 		sq: SlowQuery{
-			Timestamp: t2,
-			UnixTime:  1459470669,
-			Query:     "SELECT * FROM (SELECT  T1.orderNumber,  STATUS,  SUM(quantityOrdered * priceEach) AS  total FROM orders WHERE total > 1000 AS T1 INNER JOIN orderdetails AS T2 ON T1.orderNumber = T2.orderNumber GROUP BY  orderNumber) T WHERE total > 100;",
-			//NormalizedQuery: "select * from (select t1.ordernumber, status, sum(quantityordered * priceeach) as total from orders where total > ? as t1 inner join orderdetails as t2 on t1.ordernumber = t2.ordernumber group by ordernumber) t where total > ?;",
+			Timestamp:       t2,
+			UnixTime:        1459470669,
+			Query:           "SELECT * FROM (SELECT  T1.orderNumber,  STATUS,  SUM(quantityOrdered * priceEach) AS  total FROM orders WHERE total > 1000 AS T1 INNER JOIN orderdetails AS T2 ON T1.orderNumber = T2.orderNumber GROUP BY  orderNumber) T WHERE total > 100",
+			NormalizedQuery: "select * from (select t1.ordernumber, status, sum(quantityordered * priceeach) as total from orders where total > ? as t1 inner join orderdetails as t2 on t1.ordernumber = t2.ordernumber group by ordernumber) t where total > ?",
+		},
+	},
+	{
+		rawE: rawEvent{
+			lines: []string{
+				"# Time: not-a-parsable-time-stampZ",
+				"SET timestamp=1459470669;",
+				"SELECT * FROM orders WHERE total > 1000;",
+			},
+		},
+		sq: SlowQuery{
+			Timestamp:       t2,
+			UnixTime:        1459470669,
+			Query:           "SELECT * FROM orders WHERE total > 1000",
+			NormalizedQuery: "select * from orders where total > ?",
+			Tables:          "orders",
+			Statement:       "select",
 		},
 	},
 	{
```

Comment:
Do you wan to insert a multi-line query to test the multi-line query joining stuff? :D